examples/AspNet/Global.asax.cs

```diff
@@ -53,6 +53,17 @@ protected void Application_Start()
                         zipkinOptions.Endpoint = new Uri(ConfigurationManager.AppSettings["ZipkinEndpoint"]);
                     });
                     break;
+                case "otlp":
+                    // Adding the OtlpExporter creates a GrpcChannel.
+                    // This switch must be set before creating a GrpcChannel/HttpClient when calling an insecure gRPC service.
+                    // See: https://docs.microsoft.com/aspnet/core/grpc/troubleshoot#call-insecure-grpc-services-with-net-core-client
+                    AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
```

Comment:
Is this relevant for asp.net?

Reply:
Yeah I believe this is to ensure that we have support for unencrypted gRPC channels.  We set this AppContext switch right before adding an OtlpExporter in other parts of the solution too.

Reply:
You're right! This is only needed for gRPC net client. I have removed it.